pkgs/tools/misc/zsjed/default.nix

```diff
@@ -0,0 +1,68 @@
+{ coreutils
+, diffutils
+, fetchurl
+, findutils
+, gawk
+, lib
+, makeWrapper
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zsjed";
+  version = "0.3";
+
+  src = fetchurl {
+    url = "https://ytrizja.de/distfiles/zsjed-${version}.tar.gz";
+    sha256 = "1rcrvrqjsv80ldz7cscqbrf98jb0yd2in97irrzlpask699z5k29";
+  };
+
+  nativeBuildInputs = [
+    diffutils
+    makeWrapper
+  ];
+
+  buildInputs = [
```

Comment:
That `buildInputs` is handled equivalent to `depsHostTarget` is outright confusing... (see also: https://nixos.wiki/wiki/Cross_Compiling)

Reply:
Rule of thumb: everything linked against goes to buildInputs and everything executed or what should be in PATH goes to nativeBuildInputs. About all other cross compile issues I would just wait until someone discovers problems while building.